arch/arm/arm-m/systick/systick.c

```diff
@@ -109,7 +109,9 @@ lk_time_t current_time(void)
     do {
         t = ticks;
         delta = (volatile uint32_t)SysTick->VAL;
+#if       (__CORTEX_M >= 0x03)
         DMB;
```

Comment:
probably better to ifdef out the definition of DMB in the header than do it on every use.